tests/components/homematicip_cloud/conftest.py

```diff
@@ -98,6 +99,8 @@ def default_mock_home_fixture(mock_connection) -> AsyncHome:
     mock_home.on_create(hap.async_create_entity)
 
     hass.data[HMIPC_DOMAIN] = {HAPID: hap}
+    hmip_config_entry.state = ENTRY_STATE_LOADED
```

Comment:
Why do we need to set state manually? That should happen when the entry is set up.

Reply:
If i only setup a default_mock_hap (homematicip accesspoint) for testing then the config_entry needs to be manually setup too. The entry is not created by ha automatically.

Reply:
I refactored the get_mock_hap method to use config_entry setup. Much nicer now.
Thanks a lot.